micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/MetricsAspect.groovy

```diff
@@ -56,5 +66,23 @@ class MetricsAspect {
                 trimLeadingCharacter(str, '.' as Character), '.' as Character)
     }
 
+    public static String sanatizeIpAddressIfPresent(String metricName) {
```

Comment:
@dst +1 for `InetAddressValidator` - e.g. 2a00:1450:400c:c02::71 is a valid IPv6 address and `ipv6Pattern` won't catch it.